packages/ui/components/input/Input.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { InputHTMLAttributes, ReactNode, forwardRef, useId } from "react";
+import { classNames } from "@documenso/lib";
+
+interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  label?: string;
+  labelSrOnly?: boolean;
+}
+
+export const Input = forwardRef<HTMLInputElement, InputProps>((props, ref) => {
+  const { label, labelSrOnly, className, ...rest } = props;
+  const id = useId();
+
+  return (
+    <div>
+      {label && (
+        <label
+          htmlFor={id}
+          className={classNames(
+            "block text-sm font-medium text-gray-700",
+            labelSrOnly && "sr-only"
+          )}>
+          {label}
+        </label>
+      )}
+      <input
+        id={id}
+        ref={ref}
+        {...rest}
+        className={classNames(
+          "focus:border-neon focus:ring-neon mt-1 block w-full rounded-md border border-gray-300 py-2 px-3 shadow-sm focus:outline-none sm:text-sm",
```

Comment:
I think it's better to have `mb-1` class on label rather then `mt-1` on input just to avoid unexpected margins if `label` is not provided

Reply:
Nice suggestion